assets/monitoring/grafana/v1alpha1/grafana.yaml

```diff
@@ -9,10 +9,9 @@ spec:
   ingress:
     enabled: False
   secrets:
-  - "{{ .scyllaDBMonitoringName }}-grafana-serving-certs"
+  - "{{ .servingCertSecretName }}"
   - "{{ .scyllaDBMonitoringName }}-prometheus-client-grafana"
   configMaps:
-  - "{{ .scyllaDBMonitoringName }}-prometheus-serving-ca"
```

Comment:
used in https://github.com/tnozicka/scylla-operator/blob/21cfc6ef1f86d4e8463d146e68263ff31faae340/assets/monitoring/grafana/v1alpha1/prometheus.grafanadatasource.yaml#L19

Reply:
done